kubectl-plugin/pkg/util/generation/generation.go

```diff
@@ -61,10 +65,19 @@ func (rayClusterSpecObject *RayClusterSpecObject) generateRayClusterSpec() *rayv
 	// TODO: Look for better workaround/fixes for RayStartParams. Currently using `WithRayStartParams()` requires
 	// a non-empty map with valid key value pairs and will not populate the field with empty/nil values. This
 	// isn't ideal as it forces the generated RayCluster yamls to use those parameters.
+	headRayStartParams := map[string]string{
+		"dashboard-host": "0.0.0.0",
+	}
+	workerRayStartParams := map[string]string{
+		"metrics-export-port": "8080",
+	}
+	maps.Copy(headRayStartParams, rayClusterSpecObject.HeadRayStartParams)
+	maps.Copy(workerRayStartParams, rayClusterSpecObject.WorkerRayStartParams)
```

Comment:
For backwards compatibility. Lmk if it's better to remove these defaults and address the TODO comment above.